learn/experimental/vector_search.mdx

```diff
@@ -37,14 +37,21 @@ To ensure proper scaling of Meilisearch Cloud's latest AI-powered search offerin
 
 ### Generate vector embeddings
 
-To use vector search, first configure the `embedders` index setting. You may configure multiple embedders for an index. 
+To use vector search, first configure the `embedders` index setting. You may configure multiple embedders for an index.
 
-Embedders generate vector data from your documents. Meilisearch natively supports [OpenAI](https://openai.com/) and [Hugging Face](https://huggingface.co/) embedders. 
+Embedders generate vector data from your documents. Meilisearch natively supports the following embedders:
 
-It is also possible to use custom embedders. In this case, you must generate the embeddings manually and include them as a field in your documents.
+- [OpenAI](https://openai.com/)
+- [Hugging Face](https://huggingface.co/)
+- [Ollama](https://ollama.com/)
+- any embedder with a REST interface
 
-#### Generate auto-embeddings with OpenAI 
+It is also possible to supply custom embeddings. In this case, you must generate the embeddings manually and include them as a field in your documents.
```

Comment:
Do you want to specify the `_vectors` field or link to somewhere that explains this, maybe?